tasks/symlink.yml

```diff
@@ -1,7 +1,16 @@
 ---
 # Symlinks shared paths
+- name: ANSISTRANO | Ensure shared paths directory exists
+  file:
+    state: directory
```

Comment:
:-1:  the shared paths represent just that, _paths_ => leading to any node in the filesystem be they files or dirs or fifos or whatnot.

Otherwise this feature should be called shared_dirs and be accompanied by a shared_files .. meh